sdk/tests/conformance2/rendering/00_test_list.txt

```diff
@@ -18,5 +18,6 @@ framebuffer-unsupported.html
 instanced-arrays.html
 --min-version 2.0.1 instanced-rendering-bug.html
 out-of-bounds-index-buffers-after-copying.html
+rendering-sampling-feedback-loop.html
```

Comment:
I agree.  Let's put this to 2.0.1. Chrome will probably be able to enforce this in the initial release, so any contents violating this rule will not run.